investigator/investigator/adviser_trigger/investigate_adviser_trigger.py

```diff
@@ -35,7 +36,7 @@
 @adviser_trigger_in_progress.track_inprogress()
 async def parse_adviser_trigger_message(adviser_trigger: AdviserTriggerMessage, openshift: OpenShift) -> None:
     """Parse adviser trigger message."""
-    await wait_for_limit(openshift)
+    await wait_for_limit(openshift, workflow_namespace=Configuration.THOTH_BACKEND_NAMESPACE)
```

Comment:
```
openshift.backend_namespace
```
We don't need any sort of configuration added here since we have the openshift instance

Reply:
> ```
> openshift.backend_namespace
> ```
> 
> We don't need any sort of configuration added here since we have the openshift instance

Isn't it required for `https://github.com/thoth-station/common/blob/fd1229305a1dcaabe51eecbb290887bd1ccdf7d5/thoth/common/workflows.py#L463` to add the workflow_namespace and therefore `wait_for_limit` method requires it as well?

Reply:
Yeah, but I'm saying since we pass the openshift instance we can pull the configuration from there rather than adding it to investigator

Reply:
Right, I guess is the same, in any case they need to be provided in DC

Reply:
do we want to change it?

Reply:
It doesn't matter to me all that much, I just know that we are duplicating configuration but that might in fact make it more obvious what needs to be done for investigator deployment.

Reply:
Thanks!!